x-pack/plugins/security_solution/public/common/components/exceptions/viewer/exception_entries.tsx

```diff
@@ -134,7 +134,7 @@ const ExceptionEntriesComponent = ({
             </EuiFlexItem>
           </EuiFlexGroup>
         </EuiFlexItem>
-        <EuiFlexItem>
+        <EuiFlexItem grow={false}>
```

Comment:
What caused this change, here? I couldn't see it having any effect.

Reply:
It was a matter of the buttons floating to the the top of the container when the left description list was long and the number of exceptions were not. 

Here's a before:
![Screen Shot 2020-06-09 at 6 50 39 PM](https://user-images.githubusercontent.com/10927944/84209723-dd188c80-aa84-11ea-83dd-ea577f267600.png)

And after:
![Screen Shot 2020-06-09 at 9 48 46 AM](https://user-images.githubusercontent.com/10927944/84209735-e4d83100-aa84-11ea-91b2-208a6b3a49c2.png)